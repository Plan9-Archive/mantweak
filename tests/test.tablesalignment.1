#!/bin/rc
if(result=`{cmp <{cat $0.in | spacewidths } <{cat $0.out}}){}
if not {
	echo $0 failed: $"result;
	exit "fail";
}
