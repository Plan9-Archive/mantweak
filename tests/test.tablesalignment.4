#!/bin/rc
if(result=`{cmp -L <{cat $0.in | mantweak -l 2 -n -t 4 } <{cat $0.out}}){}
if not {
	echo $0 failed: $"result;
	exit "fail";
}
