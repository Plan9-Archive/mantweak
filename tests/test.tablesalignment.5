#!/bin/rc
font=/lib/font/bit/dejavusans/unicode.18.font
if(result=`{cmp -L <{cat $0.in | mantweak -l 2 -n -t 4 } <{cat $0.out}}){}
if not {
	echo $0 failed: $"result;
	exit "fail";
}
