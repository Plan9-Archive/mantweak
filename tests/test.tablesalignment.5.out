DESCRIPTION

Factotum is a user-level file system that acts as the
authentication agent for a user.  It does so by managing a
set of keys. A key is a collection of information used to
authenticate a particular action.  Stored as a list of
attribute=value pairs, a key typically contains a user, an
authentication domain, a protocol, and some secret data.

The RPC Protocol
  Authentication is performed by

	1)	opening rpc

	2)	setting up the protocol and key to be used (see the
		start RPC below),

	3)	shuttling messages back and forth between factotum and
		the other party (see the read and write RPC's) until
		done

	4)	if successful, reading back an AuthInfo structure (see
		authsrv(2)).

  The RPC protocol is normally embodied by one of the routines
  in auth(2). We describe it here should anyone want to extend
  the library.
